components/BB.js

```diff
@@ -77,10 +101,21 @@ export class BB extends Component {
   }
   scrollToNextSteps() {
     window.location = "#next-steps";
+    const maxMobile = parseFloat(globalTheme.max.xs);
+    window.screen.width < maxMobile ? window.scrollBy(0, -90) : null;
```

Comment:
I tested with my phone and this offset is working great! Can you apply it to the favourites page too?

Reply:
Oh yes, I meant to not forget about that...